### PR TITLE
feat: LT-2359 - Multiple control list entries

### PR DESCRIPTION
**Pull request by jfaracik**
**What's new**
* Now possible to add multiple control list entries throughout LITE
* Click a control list entry to view its details

### CODE REVIEW

File: cases/forms/review_goods_clc.py [hunk 1]
```diff
@@ -19,23 +19,20 @@ def review_goods_clc_query_form(request, back_url, is_goods_type):
...
-                                inset_text=False,
                             ),
                         ],
                     ),
                     Option(key=conditional(is_goods_type, "False", "no"), value="No"),
                 ],
             ),
             RadioButtons(
-                title="Which report summary would you like to use? (optional)",
+                title="Which report summary would you like to use?",
```
Comment (BenAndrew-Kainos) [inline cases/forms/review_goods_clc.py:32]: Content for this page

File: cases/services.py [hunk 2]
```diff
@@ -299,7 +303,7 @@ def get_good(request, pk):
 
 def get_goods_type(request, pk):
     data = get(request, GOODS_TYPE_URL + pk)
-    return data.json(), data.status_code
+    return {"good": data.json()}, data.status_code
```
Comment (BenAndrew-Kainos) [inline cases/services.py:306]: Why are you doing this?

Comment (jfaracik) [reply cases/services.py:311]:
```
API doesn't structure the endpoints in a way that flags (currently) works, so wrap data in dictionary
```

I've added a comment

File: cases/forms/review_goods_clc.py (hunk 1 above)
Comment (jfaracik) [reply cases/forms/review_goods_clc.py]: Done

File: cases/services.py (hunk 2 above)
Comment (BenAndrew-Kainos) [reply cases/services.py:313]: I feel like we should fix that on the API. For now it's fine though

**Pull request merged**